apps/services/sessions/src/app/sessions/sessions.service.ts

```diff
@@ -97,8 +98,9 @@ export class SessionsService {
 
     return this.sessionModel.create({
       ...rest,
+      userAgent: userAgent.substring(0, USER_AGENT_MAX_LENGTH),
       sessionId: sid,
-      device: this.formatUserAgent(session.userAgent),
+      device: this.formatUserAgent(userAgent),
```

Comment:
The `device` column is also of type `string`. Do we need to make sure `formatUserAgent` never exceeds the max length (default length 255)?

Reply:
No I don't think we need to worry about that
the format function is 
```
private formatUserAgent(userAgent: string): string | undefined {
    const ua = uaParser(userAgent)
    const browser = ua.browser.name || ''
    const os = ua.os.name || ''

    return browser || os
      ? `${browser}${browser && os ? ` (${os})` : os}`
      : undefined
  }
```
And the combination of browers name and os name are from ua-parser-js
https://www.npmjs.com/package/ua-parser-js